routes/auth.js

```diff
@@ -12,28 +12,27 @@ router.post("/register", async (req, res) => {
   if (error) return res.status(400).send(error.details[0].message);
 
   // Checks if user is already in the database
-  let user = await User.findOne({ email: req.body.email });
-  if (user)
+  let _user = await user.findUserByEmail(req.body.email);
```

Comment:
even I'm not a fan of this user name 🙈, let me know if you have any better suggestion @dxtrlws

Reply:
I'm fine with calling it whatever you want. As long as it's descriptive to what's being stored in the variable.